core/controllers/acl_decorators.py

```diff
@@ -2638,6 +2639,8 @@ def test_can_edit_entity(self, entity_type, entity_id, **kwargs):
             return can_edit_skill(reduced_handler)(self, entity_id, **kwargs)
         elif entity_type == feconf.ENTITY_TYPE_STORY:
             return can_edit_story(reduced_handler)(self, entity_id, **kwargs)
+        elif entity_type == constants.TEMP_SKILL_ENTITY:
```

Comment:
Also no -- again it is not an entity. You need a different handler/decorator altogether, or possibly a separate arg. Otherwise, you're combining multiple disparate things under the same concept.

Reply:
Done, removed this.